Fig3_PyGMT_backgrounds.py

```diff
@@ -0,0 +1,44 @@
+import pygmt
+
+region = [-25, -13, 63.2, 66.7]
+
+fig = pygmt.Figure()
+with fig.subplot(
+    nrows=2,
+    ncols=2,
+    subsize=("12c", "8.5c"),
+    autolabel="(a)+o0.15c/0.3c+gwhite@30+v",  # vertical order of labels

```

Comment:
Remove unnecessary comments
```suggestion
    autolabel="(a)+o0.15c/0.3c+gwhite@30+v",
```